test/uk/gov/hmrc/plasticpackagingtax/registration/views/partials/PptHelplineDetailsSpec.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.plasticpackagingtax.registration.views.partials
+
+import base.unit.UnitViewSpec
+import org.scalatest.matchers.must.Matchers
+import uk.gov.hmrc.plasticpackagingtax.registration.views.html.partials.pptHelplineDetails
+
+class PptHelplineDetailsSpec extends UnitViewSpec with Matchers {
+
+  private val pptHelplinePartial = instanceOf[pptHelplineDetails]
```

Comment:
Is this component used anywhere?

Reply:
It is not but anticipate its return at some point once the necessary PPT support has been organised.